src/main/webapp/js/pages/savedPlanners.js

```diff
@@ -66,6 +84,22 @@ function createCard(planName, tableData, creditsData) {
   document.getElementById('plan-column').appendChild(card);
 }
 
+/**
+ * Tells the server to delete the plan.
+ * @param {Number} id The id of the plan to delete.
+ */
+function deletePlan(id) {
+  const params = new URLSearchParams();
+  params.append('id', id);
+  try {
+    fetch('/delete-plan', {method: 'POST', body: params});
```

Comment:
you'll want to check if the response is `.ok`, and put up an error if it is not.